types/bull/index.d.ts

```diff
@@ -755,6 +818,38 @@ declare namespace Bull {
      * Array of Redis clients the queue uses
      */
     clients: Redis.Redis[];
+
+    /**
+     * Set clientName to Redis.client
+     * https://github.com/OptimalBits/bull/blob/4b3ba1a75a7d29263897f91bd943298e0e6dbba4/lib/worker.js#L12
+     */
+    setWorkerName(): Promise<any>;
+
+    /*
```

Comment:
This and all the following comments start with `/*` but not `/**`, which makes them just plain comments, not JSDoc. Also, nitpick: there’s no need to include type info in the param tag; since these are TypeScript type definitions, it’s redundant:

```diff
- @params list: string String with all redis clients
+ @param list String with all redis clients
```

Reply:
I think it's better to delete code links and redundant type info in comments.
it would also be good to modify comments start with ```/*``` to ```/**```
Thanks for comments!